frontend/src/routes/Governance/components/ViewDiffApiCall.tsx

```diff
@@ -62,13 +63,27 @@ export function ViewDiffApiCall({ item }: Readonly<{ item: ResultsTableData }>)
     }
   }, [isModalOpen, kind, apiVersion, templateName, cluster, t])
 
+  const isDiffEmpty = !relatedObjs?.some((obj: any) => obj.properties?.diff)
```

Comment:
I had to rethink lol so can we use 
```suggestion
  const isAllDiffEmpty = relatedObjs?.every((obj: any) => !obj.properties?.diff)
```